mlflow/tracking/context/databricks_job_context.py

```diff
@@ -10,6 +10,13 @@
     MLFLOW_DATABRICKS_JOB_TYPE,
     MLFLOW_DATABRICKS_WORKSPACE_URL,
     MLFLOW_DATABRICKS_WORKSPACE_ID,
+    MLFLOW_DATABRICKS_GIT_URL,
+    MLFLOW_DATABRICKS_GIT_PROVIDER,
+    MLFLOW_DATABRICKS_GIT_COMMIT,
+    MLFLOW_DATABRICKS_GIT_RELATIVE_PATH,
+    MLFLOW_DATABRICKS_GIT_REFERENCE,
+    MLFLOW_DATABRICKS_GIT_REFERENCE_TYPE,
+    MLFLOW_DATABRICKS_GIT_STATUS,
```

Comment:
Nit on naming: `GIT_URL` or `REPO_URL`? If we're talking about Databricks Repos, I think `REPO_URL` is better. Ditto for the other names.

Reply:
The naming here was done in line with the existing field names in jobs, repos, and webapp, but we could also just make it `GIT_REPO_URL` like the name of the `CommandContext` fields.

Reply:
Updated to `GIT_REPO_URL` to make it clear.